app/modes/library/__init__.py

```
@@ -0,0 +1,5 @@
+from flask import Blueprint
+
+library_bp = Blueprint('library', __name__, template_folder='templates', static_folder='static')
+
+from . import routes  # noqa: E402
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)